consensus/src/quorum_store/proof_coordinator.rs

```diff
@@ -138,58 +115,48 @@ impl ProofCoordinator {
         validator_verifier: &ValidatorVerifier,
     ) -> Result<Option<ProofOfStore>, SignedDigestError> {
         if !self.digest_to_proof.contains_key(&signed_digest.digest()) {
-            return Err(SignedDigestError::WrongInfo);
+            if signed_digest.info().batch_author == self.peer_id {
```

Comment:
We should figure out the DDoS protection here. Previously we called init before sending out the messages and only accept existing digests. Now anybody can send garbage to us and we will store it in memory. 
The discussed fix: assume local speed is much faster than a round trip and check if we have the digest in batch_store.

Reply:
hm, but doesn't that (almost) guarantee that we should have our own signature? Maybe we should change that to error log?